pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/auth_proxy_client_role.go

```diff
@@ -43,6 +44,13 @@ func (f *AuthProxyClientRole) SetTemplateDefaults() error {
 const clientClusterRoleTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: metrics-reader
+    app.kubernetes.io/component: kube-rbac-proxy
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kubernetes.io/managed-by: k8s.io
```

Comment:
```suggestion
    app.kubernetes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02